api/src/main/java/javax/enterprise/context/spi/ActiveContext.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package javax.enterprise.context.spi;
+
+import javax.enterprise.inject.Instance;
+import java.lang.annotation.Annotation;
+
+/**
+ * Representation of a context which has been activated
+ *
+ * @author John D. Ament
+ */
+public interface ActiveContext {
```

Comment:
I'll take a shot in the dark - _ManagedContext_ ? Since you can actually manage its lifecycle yourself (or the interceptor can manage it for you).
Another one could be _ActivableContext_. (granted, _activable_ is a bit a of a neologism :-) )
Since it does not really need to be "active" at all times I would try to refrain from using word "active" in its name. Which would instantly make a call to `isActive()` sound meaningful.